libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -35,7 +36,7 @@ var (
 	HugePageSizes, _ = cgroups.GetHugePageSize()
 )
 
-var errSubsystemDoesNotExist = errors.New("cgroup: subsystem does not exist")
+var errSubsystemDoesNotExist = fmt.Errorf("cgroup: subsystem does not exist")
```

Comment:
why?

Reply:
We have a `github.com/pkg/errors` import, so we can't import `errors` (without renaming and that's just ugly). I could do `errors.Errorf` but then you'd get a stack trace in the package which isn't really useful. `fmt.Errorf` is otherwise essentially the same as `errors.New`.

Reply:
Why not use https://godoc.org/github.com/pkg/errors#New

Reply:
> New also records the stack trace at the point it was called.